src/orion/algo/random.py

```diff
@@ -64,3 +64,16 @@ def suggest(self, num):
                 points.append(new_point)
 
         return points
+
+
+    @contextmanager
+    def warm_start_mode(self):
+        # NOTE: The Random algorithm just discards any points observed while in
+        # 'warm-start mode':
+        backup = self._trials_info
+        self._trials_info = self._trials_info.copy()
```

Comment:
We could avoid the copy here since they won't be used anyway.

Reply:
Not exactly: One of either `backup` or `self._trials_info` needs to be copied.